blocks/header/header.css

```diff
@@ -838,19 +848,20 @@ header nav .nav-tools {
   }
 
   header .nav-menu-overlay .nav-menu-column.right > .submenu > li {
-   width: calc(50% - 20px);
+    width: calc(45vw - 4vw);
```

Comment:
Can be 41vw right?

Reply:
yes. I was trying with different metrics. I will update

Reply:
forgot to remove calc